features/FEATURE_LWIP/lwip-interface/mbed_lib.json

```diff
@@ -72,6 +72,14 @@
             "help": "Maximum number of open UDPSocket instances allowed, including one used internally for DNS.  Each requires 84 bytes of pre-allocated RAM",
             "value": 4
         },
+        "pbuf-pool-size": {
+            "help": "Number of pbufs in pool - usually used for received packets, so this determines how much data can be buffered between reception and the application reading. If a driver uses PBUF_RAM for reception, less pool may be needed",
+            "value": 16
```

Comment:
Oh. Actually this isn't an Odin specific change. Can we restrict this change to just the Odin board for now? That would help minimize the impact to applications above us.

We probably shouldn't increase this value globally unless it's a minor release. That's just asking for trouble.